app/assets/javascripts/components/Map.js

```diff
@@ -75,10 +75,12 @@ class Map extends React.Component {
   }
 
   _popUpFor(layer) {
+    // debugger
     return `<h1>${layer.name}</h1>
             <p>${layer.description}</p>
             <div class="link-group">
-              <a href="/tasks/${layer.id}">Learn more</a>
+              <a href="/campaigns/${layer.events ? layer.events[0].campaign_id : this.props.campaignId}">Campaign</a>
```

Comment:
A task might belong to more than one event and/or campaigns, so I think we should maybe iterate over them and display a link to each. Probably worth checking with Juanca for ideas on how best to show it.